src/tiff_decoder.ts

```diff
@@ -289,19 +289,22 @@ export default class TIFFDecoder extends IOBuffer {
     // General Image Dimensions
     const width = ifd.width;
     const height = ifd.height;
+    const rowsPerStrip = ifd.rowsPerStrip || 2 ** 32 - 1;
```

Comment:
If the default value is generally meaningful (or written in the specification), we put it here: https://github.com/image-js/tiff/blob/67169f92bbb935dc142ecc620de9350a07aae1dd/src/tiff_ifd.ts#L94